furya/chain.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "furya",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://furya.xyz/",
+  "pretty_name": "furya",
+  "chain_id": "furya-1",
+  "daemon_name": "furyad",
+  "node_home": "$HOME/.furyad",
+  "bech32_prefix": "furya",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md